About-Page/about.html

```diff
@@ -0,0 +1,160 @@
+          world-class education. We offer free online courses offers free online
+          education courses . And under each category we provide , there are
+          numerous subjects for almost every remote career.
+        </p>
+        <p class="contact-message">
+          Pleade contact me fot more information about the program
+        </p>
+        <span><a href="">remote.experts@gmail.com</a></span>
+      </div>
+    </section>
+
+    <section class="logo-info">
+      <div class="container">
+        <span class="logo-title">"Hello Job seekers Land your Remote Job"</span>
+        <h2 class="logo-heading">
+          JOIN OUR CREATIVE COMMONS GLOBAL SUMMIY 2015
+        </h2>
+        <p class="contact-message">
+          Remote Experts was decided through the logo competition from 8. June
+          to 7. July.
+        </p>
+        <div class="logo">
+          <svg class="icon">
+            <use xlink:href="../img/sprite.svg#remote-work"></use>
+          </svg>
+          <p>REMOTE EXPERTS</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="clients">
+      <div class="container">
+        <h2 class="clients-heading">Client from privious conferences</h2>
+        <p class="previous-clients">
+          Take a look at our students who already finished the program, and see
+          what they had to say about the program.
+        </p>
+        <div class="clients-image">
+          <div class="client-img1">
+            <div class="red-filter">
+              2015 <br />
+              Remote experts 2015 conferences
+            </div>
+          </div>
+          <div class="client-img2">
+            <div class="red-filter">
+              2017 <br />
+              Remote experts 2017 conferences
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="partner-section">
+      <div class="partner-section__header">
+        <h2 class="partner-section__title">featured Speakers</h2>
+        <div class="red-line"></div>
+      </div>
+      <div class="partner-section__body">
+        <ul class="partner-section__list">
+          <li class="partner-section__list-item">
+            <i class="fab fa-google" aria-hidden="true">oogle</i>
+          </li>
+          <li class="partner-section__list-item">
+            <i class="fab fa-airbnb" aria-hidden="true"> airbnb</i>
+          </li>
+          <li class="partner-section__list-item">
+            <i class="fab fa-microsoft" aria-hidden="true"> Microsoft</i>
+          </li>
+          <li class="partner-section__list-item">
+            <i class="fab fa-apple" aria-hidden="true"> Apple</i>
+          </li>
+          <li class="partner-section__list-item">
+            <i class="fab fa-free-code-camp" aria-hidden="true"></i>
+            freeCodeCamp
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <footer class="footer-section">
```

Comment:
In the Behance design, for the About page, the background is a shade of black.

- [ ] Kindly, redesign your footer from the About page in order to have that shade of black that the Behance design is using.
Your footer:
![image](https://user-images.githubusercontent.com/83596135/157766336-b5b0e316-6079-4615-b3ab-0f2107a101ca.png)
Behance design:
![image](https://user-images.githubusercontent.com/83596135/157766430-e80118d1-53b3-44e5-9695-5f06d3cf71e9.png)

**Note:** for the mobile version, please ensure to keep the white background for both Home and About pages.